controls/V-73285.rb

```diff
@@ -69,71 +69,70 @@
   Local accounts can be configured to expire with the command Net user
   [username] /expires:[mm/dd/yyyy], where username is the name of the temporary
   user account."
-  emergency_account = input('emergency_account')
-  if !emergency_account.empty?
 
-    emergency_account.each do |user|
+  domain_role = command('wmic computersystem get domainrole | Findstr /v DomainRole').stdout.strip
+  emergency_accounts_list = input('emergency_accounts')
+  emergency_accounts_data = []
```

Comment:
Reviewed several controls to address #4 and standardized V-73285 using the 2019 approach